platform/app/src/routes/Mode/Mode.tsx

```diff
@@ -164,7 +164,7 @@ export default function ModeRoute({
 
   // Preserve the old array interface for hotkeys
   const hotkeys = Array.isArray(hotkeyObj) ? hotkeyObj : hotkeyObj?.hotkeys;
-  const hotkeyName = hotkeyObj?.name || 'hotkey-definitions-v2';
+  const hotkeyName = hotkeyObj?.name || 'hotkey-definitions';
```

Comment:
Does this affect anyone migrating from v2?

Reply:
I don't think that was supposed to be v2, prob just a typo